docs/pages/versions/unversioned/distribution/app-signing.md

```diff
@@ -2,10 +2,49 @@
 title: App signing
 ---
 
-Process is automated for iOS and Android, but in both cases you can choose to provide your own overrides. Both `expo build:ios` and
+Expo automates the process of signing your app for iOS and Android, but in both cases you can choose to provide your own overrides. Both `expo build:ios` and
 `expo build:android` commands generate signed applications ready to be uploaded into respective stores.
 
-## Android 
+On this page, we'll talk about the credentials that each platform requires. If you're curious about how we store your credentials on our end, take a look at our [security documentation](../security/).
+
+## iOS
+
+The 3 primary iOS credentials, all of which are associated with your Apple Developer account, are:
+
+- Distribution Certificate
+- Provisioning Profiles
+- Push Notification Keys
+
+Whether you let Expo handle all your credentials, or you handle them all yourself, it's smart to understand what each of these credentials mean, when and where they're used, and what happens when they expire or are revoked.
+
+### Distribution Certificate
+
+Whereas the rest of these credentials are about the apps themselves, the certificate is all about you, the developer. You may only have one distribution certificate associated with your Apple Developer account.
+This certificate will be used for all of your apps. If this certificate expires, your apps in production will not be affected. However, you will need to generate a new certificate if you want to upload new apps to the app store, or update any of your existing apps. Deleting a distribution certificate has no effect on any apps already on the App Store.
+
+### Push Notification Keys
+
+Apple Push Notification Keys, or APN keys, allow the associated apps to send and receive push notifications. By default, any app built with Expo will require an APN key. This is so that you can enable push notifications for your app through a quick [OTA update](../../guides/configuring-ota-updates/), rather than needing to submit an entirely new binary.
```

Comment:
```suggestion
Apple Push Notification Keys (often abbreviated as APN keys) allow the associated apps to send and receive push notifications. By default, any app built with Expo will require an APN key. This is so that you can enable push notifications for your app through a quick [OTA update](../../guides/configuring-ota-updates/), rather than needing to submit an entirely new binary.
```